src/main/scala/is/hail/annotations/StagedRegionValueBuilder.scala

```diff
@@ -88,7 +88,8 @@ class StagedRegionValueBuilder private(val fb: FunctionBuilder[_], val typ: Type
     else
       startOffset.store(pOffset)
     assert(staticIdx == 0)
-    c = Code(c, elementsOffset.store(startOffset + t.byteOffsets(0)))
+    if (!typ.isOfType(TStruct.empty()))
```

Comment:
Slightly clearer if the condition matches the analogous one in advance, comparing against `t.size`.